scripts/build.js

```diff
@@ -1,42 +1,38 @@
-const fs = require('fs')
-const rp = require('request-promise-native')
-const xlsx = require('xlsx')
+const fs = require('fs');
+const rp = require('request-promise-native');
+const xlsx = require('xlsx');
 
-const url = 'http://www.iso15022.org/MIC/ISO10383_MIC.xls'
-const encoding = null
+const url = 'http://www.iso15022.org/MIC/ISO10383_MIC.xls';
+const encoding = null;
 
 function exit(err) {
   if (err) {
     /* eslint-disable no-console */
-    console.log(err)
+    console.log(err);
     /* eslint-enable no-console */
-    process.exit(1)
+    process.exit(1);
   } else {
-    process.exit(0)
+    process.exit(0);
   }
 }
 
 rp({ url, encoding })
   .then(res => {
-    try {
```

Comment:
Throwing inside of a promise (which we are doing here since we're in a then) will reject that promise, so even if these 2 throw, the catch at the end will call exit.